apps/mobile/eas.json

```diff
@@ -22,6 +22,9 @@
       "ios": {
         "simulator": true,
         "cocoapods": "1.15.2"
+      },
+      "android": {
+        "image": "latest"
```

Comment:
Good question, and I'm not totally sure what to use. The obvious choice is what is in `build.gradle` which is `ndkVersion = "26.1.10909125"` though since it's defined there I don't see the advantage to adding it to the `eas.json`

Reply:
As far as I can tell we don't need to add this. But I'll keep an eye on it and if things start getting flaky might be worth adding.